src/io/output_mesh.hh

```diff
@@ -72,18 +63,26 @@ public:
     /// Gives iterator to the LAST element of the output mesh.
     OutputElementIterator end();
     
-    /// Selects the error control field out of output field set according to input record.
-    void select_error_control_field(FieldSet &output_fields);
-    
+    /// Creates nodes_, connectivity_ and offsets_ data caches.
+    void create_data_caches();
```

Comment:
Either describe and rename e.g. create_dummy_output_mesh, or just make empty arrays in constructor and rewrite them latter.